app/src/main/java/ch/heigvd/pro/b04/android/Poll/PollAdapter.java

```diff
@@ -91,29 +106,33 @@ private void bindQuestion(Question question, boolean answered) {
                 return new HeaderViewHolder(parent, state, lifecycleOwner);
             case VIEW_TYPE_QUESTION:
                 return new QuestionViewHolder(parent);
+            case VIEW_TYPE_INSTRUCTION:
+                return new InstructionViewHolder(parent);
             default:
                 throw new IllegalStateException("Unknown view type.");
         }
     }
 
     @Override
     public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int position) {
-        if(position != 0) {
-            Question q = questions.get(position-1);
+        if(position != 0 && position != 1) {
+            Question q = questions.get(position-2);
             ((QuestionViewHolder) holder).bindQuestion(q, q.answered() );
         }
     }
 
     @Override
     public int getItemCount() {
-        return questions.size() + 1;
+        return questions.size() + 2;
     }
 
     @Override
     public int getItemViewType(int position) {
         return position == 0
                 ? VIEW_TYPE_HEADER
-                : VIEW_TYPE_QUESTION;
+                : (position == 1
+                    ? VIEW_TYPE_INSTRUCTION
+                    : VIEW_TYPE_QUESTION);
```

Comment:
Change to switch as in `EmojiGridAdapter`

Reply:
Done